charts/intel-qat-resource-driver/templates/nfd.yaml

```
@@ -0,0 +1,28 @@
+apiVersion: nfd.k8s-sigs.io/v1alpha1
+kind: NodeFeatureRule
+metadata:
+  name: intel-qat-device-rule
+  namespace: {{ include "intel-qat-resource-driver.namespace" . }}
+spec:
+  rules:
+    - name: "intel.qat"
+      labels:
+        "feature.node.kubernetes.io/qat": "true"
```

You could probably lose the quotes here?
```suggestion
        feature.node.kubernetes.io/qat: "true"
```